paymentsheet/build.gradle

```diff
@@ -134,7 +134,7 @@ android {
     }
 
     lintOptions {
-        disable "UnrememberedMutableState"
+        disable "UnrememberedMutableState", "InvalidVectorPath", "MissingDefaultResource"
```

Comment:
Why did these become necessary?

Reply:
Linter was complaining with Google's new drawable assets. I should be able to remove `MissingDefaultResource` now that we will no longer rely on device theming